### DISCUSSION TITLE
frozen page & flash out on button click in Swell

### DISCUSSION THREAD

**Discussion by shuang807** [swellstores/community#207] [Help]
Hi, 

I'm trying to use Swell. But I have got some errors. May anyone help? 

1) When I go to "Admin -> Storefronts" - The page only keeps loading with the message "We are migrating your storefront configs. Please hold..., I tried with products and without anything.., re-login, etc, but none of the actions helped.
2) I tried to delete the store, but I didn't find where to do that. How to delete a store? 
3) "Add a new store" doesn't seem to work... The button opened a window, then flashed out.

Thank you in advance!

**Comment by SusannaSwell** [comment]
hello @shuang807 , could you please share your store id, so we could check this?
